Makefile

```diff
@@ -104,10 +109,10 @@ install: conda
 	conda-forge::python-ldap=3.2.0 \
 	bioconda::rabix-bunny=1.0.4 \
 	conda-forge::jq
-	pip install -r requirements-cli.txt
 	pip install -r requirements.txt
 	pip install -r requirements-dev.txt
-	# pip install git+https://github.com/mskcc/beagle_cli.git@develop
+# pip install git+https://github.com/mskcc/beagle_cli.git@develop
+# pip install -r requirements-cli.txt # <- what happened to this file?
```

Comment:
Yes Adrian and I have also had to comment this line out, not sure where it went though